occurrences/admin.py

```diff
@@ -246,7 +249,7 @@ class VenueCustomDataAdmin(TranslatableAdmin):
 
 
 @admin.register(PalvelutarjotinEvent)
-class PalvelutarjotinEventAdmin(admin.ModelAdmin):
+class PalvelutarjotinEventAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
```

Comment:
As PalvelutarjotinEventAdmin's list_display contains get_contact_email which shows GDPR data (i.e. an email) wouldn't it make sense to log the list views because of this?
```suggestion
class PalvelutarjotinEventAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
    enable_list_view_audit_logging = True
```

Reply:
it would mean that every time an event card is shown in the UI, something would be written in the logs. Even then when using public UI and the user is not even authenticated.

Reply:
Another option would be to remove the email column (and other gdpr columns) from the list view.

Reply:
This contact information is also public information that is shown also in the public UI's event cards and details. Users are anonymous (not authenticated) when they are using the public UI. I would not like to spam the auditlog only because of this public information.

Reply:
How about masking the email column?

Reply:
> This contact information is also public information that is shown also in the public UI's event cards and details.

That **functionality** seems to be **bugged** at the moment i.e. they seem to be never shown. I looked into it a bit and the even when a contact person is set it comes through as null in the event graphql query, which makes the <OrganisationInfo> component not show the contact details.

But, the contact person's phone number and email at least can be seen publicly through the data fetch. Because the contact person's phone number and email are currently publicly available and the contact person's name would also be if the functionality worked I'd say this is ok not to add the list view audit logging here.

Reply:
Created ticket [PT-1898](https://helsinkisolutionoffice.atlassian.net/browse/PT-1898) related to this.

Reply:
After discussion this point can be left as is, or masked, if wanted to, and can be possibly iterated on later (not in this PR).